Lib/idlelib/editor.py

```diff
@@ -451,6 +451,11 @@ def update_menu_label(self, menu, index, label):
         menuitem = self.menudict[menu]
         menuitem.entryconfig(index, label=label)
 
+    def update_menu_state(self, menu, index, state):
+        "Update state for menu item at index ."
```

Comment:
I don't think there should be a space after "index" (same goes for the docstring of the recently-added `update_menu_label()` just above).